admin/workspace-providers.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Workspace Providers
+description: Learn how workspace providers can minimize latency for users.
+state: beta
+---
+
+Coder allows you to add to your deployment multiple Kubernetes clusters. These
+additional clusters are called **providers**. This can be helpful for
+distributed teams, since users can choose the provider closest to them to
+minimize latency.
+
+## Enabling Workspace Providers
+
+To enable the Workspace Providers UI elements:
+
+1. In the Coder UI, click on your avatar.
+2. In the menu that appears, click **Feature Preview**.
+3. Select **Resource Pools UI** and click **Enable**.
```

Comment:
This will not be released in 1.16 - the feature flag will still be internal only